### DISCUSSION TITLE
How is this even 100% match and top 1 on target #84?

### DISCUSSION THREAD

**Discussion by SuperQAxClub** [cssbattle/forum#128] [General]
Is this another cheat from Vincent De Oliveira?
![Screen Shot 2021-10-10 at 09 58 28](https://user-images.githubusercontent.com/7949762/136679806-6bca581f-f9fb-4423-b409-4d74d088610f.png)

**Comment by SelenIT** [comment]
The backend of CSSBattle uses Chrome's engine to check solutions, so it's better to use Chrome for playing. Some solutions use some new or experimental CSS features that are currently supported only by Chrome. There are also some "hacky" solutions also rely on platform-specific rendering of Unicode characters etc., but this one is not of that kind. It only uses a standard CSS feature that is not supported by Safari yet.

**Reply by SuperQAxClub** [reply]
I used Chrome and it still displays like that.

**Reply by SelenIT** [reply]
Aren't you using iOS/iPadOS? The rendering on your screenshot looks pretty much like Safari rendering. And, AFAIK, all browsers on iOS actually share the same Safari engine under the hood :(. If that's the case, you can use an external previewer (like @alexzaworski's [codepen](https://codepen.io/alexzaworski/pen/vwmNKr)) to check solutions.

**Reply by SuperQAxClub** [reply]
I am using M1 Mac mini

**Reply by SelenIT** [reply]
Very strange. Just checked on Chrome 94 on M1 MacBook Air (build for M1 processor). It displayed that solution as expected, identical to the reference image.

**Reply by SuperQAxClub** [reply]
Ok I just checked again and it displays as expected. It's weird that the last time I checked on safari it has that bug, then I immediately move to chrome and it also had the same bug. And now i checked again it displayed correctly

**Comment by rafehimself** [comment]
I think using any characters or unicode in html part should be ban as valid solution, overall it's CSS battle,
also some of the "top" solutions are rendered terribly, should get -100 points for style 🤣 
![image](https://user-images.githubusercontent.com/58576132/136690031-ba3ba0ec-6004-4e77-9860-0fc8dce77eb1.png)
the difference is clear for a human eye and that solution is simply ugly

**Reply by SelenIT** [reply]
The same gradient-based solutions don't look that ugly on Retina displays though, gradient edges are quite smooth there. If CSSBattle doesn't strictly require the solutions to be rendered ideally across browsers, should it be that demanding to ideal rendering across device capacities?

**Reply by gunnarbittersmann** [reply]
> I think using any characters or unicode in html part should be ban as valid solution, overall it's CSS battle

We’ve been there. Non-ASCII characters were not allowed. You could escape them using only ASCII characters like `&#x2318;` for `⌘` (which doesn’t work on [target #63](https://cssbattle.dev/play/63), unfortunately 😆), but that resulted in longer code.
However, some smart people found a way to omit that restriction and use non-ASCII characters. Then Konstantin @spalt08 came and posted a JavaScript one-liner that made this available to everybody.
Eventually, all characters became allowed. You can’t ban what you can’t control.

**Reply by Beowolve** [reply]
> > I think using any characters or unicode in html part should be ban as valid solution, overall it's CSS battle
> 
> We’ve been there. Non-ASCII characters were not allowed. You could escape them using only ASCII characters like `&#x2318;` for `⌘` (which doesn’t work on [target #63](https://cssbattle.dev/play/63), unfortunately 😆), but that resulted in longer code. However, some smart people found a way to omit that restriction and use non-ASCII characters. Then Konstantin came and posted a JavaScript one-liner that made this available to everybody. Eventually, all characters became allowed. You can’t ban what you can’t control.

I tried that char also, ~90% fit was my best fit so far :)

**Reply by gunnarbittersmann** [reply]
94.2% 🤣 https://github.com/cssbattle/forum/discussions/2#discussioncomment-488436
